presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -585,10 +585,12 @@ Procedures
 
 .. _register_partition:
 
-* ``system.register_partition(schema_name, table_name, partition_columns, partition_values, location)``
+* ``system.register_partition(schema_name, table_name, partition_columns, partition_values, optional location)``
 
   Registers existing location as a new partition in the metastore for the specified table.
 
+  Partition location is constructed using partition_columns and partition_values when location argument is ommitted.
```

Comment:
Maybe it would worthy to have an example here?

Reply:
Yes ... also use double backtick around code values..